modules/ossm-cert-manager-istio-csr-inplace-update-strategy.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/install/ossm-cert-manager-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="inplace-istio-csr-installation_{context}"]
+= istio-csr installation: InPlace update strategy
+
+The default for `spec.updateStrategy` for the `Istio` resource is `InPlace.` Follow this procedure if you going to leave `spec.updateStrategy` as `InPlace` when you create and install your `Istio` resource.
+
+.Procedure
+
+. Add the jetstack charts to your local Helm repo:
```

Comment:
```suggestion
. Add the jetstack charts to your local Helm repo by running the following command:
```
For accessibility purposes, it's helpful to include the text "by running the following command" to help orient people who are using tools like screen readers to access this content.